### DISCUSSION TITLE
<title>我的 2FA 验证无法正常使用

### DISCUSSION THREAD

**Discussion by kongxia114** [community/community#87192] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

我正常保存了我的 2FA 秘钥 在登录时总是提示验证失败

**Comment by oliver-lou** [comment]
I have exactly same problem today.I use Microsoft Authenticator,it worked.I update my Microsoft Authenticator Android App and try to update my 2FA,still don't work.

**Comment by kacphur** [comment]
I would check if you set it up correctly, my authenticator works perfect, and I would personally switch to the github one as it is a more reliable authentiactor and it's owned by themselves (github)!
